src/type_inf.jl

```diff
@@ -57,6 +57,8 @@ function infer_type_assignment_rhs(binding, state, scope)
             settype!(binding, CoreTypes.Float64)
         elseif CSTParser.isstringliteral(rhs)
             settype!(binding, CoreTypes.String)
+        elseif headof(rhs) === :TRUE || headof(rhs) === :false
```

Comment:
Why's one head `:TRUE` and the other `:false`? (C/S)hould we make those consistent?